cpp/src/slice2cpp/Gen.cpp

```diff
@@ -290,13 +290,10 @@ namespace
                 // Doxygen supports multiple syntaxes for operations, but none of them allow for a bare operation name.
                 // We opt for the syntax where operation names are qualified by what type they're defined on.
                 // See: https://www.doxygen.nl/manual/autolink.html#linkfunc.
```

Comment:
A few lines above we handle enums:

> // If a link to an enum isn't qualified (ie. the source and target are in the same module),
   // we have to place a '#' character in front, so Doxygen looks in the current scope.

We don't have this `#` for other types, does Doxygen resolve enums links different that links to other constructed types?

Reply:
Good question. What about an operation referencing another operation in the same interface, or a field referencing another field in the same struct? No `#`?

We should add a doc-comment test file that tests all these variations.